AGPs/AGP-LD1.md

```diff
@@ -0,0 +1,63 @@
+
+**The following weighted KPIs would be used to create the Aragon App Score:**
+
+| KPI | Formula | Weight |
+| -------- | -------- | -------- |
+| Utility    | Sum of (# of transactions involving an application ) | 30%  |
+| Value weighted utility    | Sum of (# of transactions involving an application * DAI value of organization's treasury) | 30%    |
+| Popularity    | Sum of (# of installations) | 10%  |
+| Value weighted Popularity    | Sum of (# of installations *  DAI value of organization's treasury) | 30% |
+
+To compute the DAI value of an organizations treasury we would use the *default vault* and compute the DAI value of ETH, ANT, and DAI held by the organization.
+
+To compute the # of transactions involving an application we would simply examine the blockchain.   
+
+For each payout cycle, there is a determined “pot” of total earnings that will get paid to apps. The top app gets paid 20% of the total pot. So, for a pot of 100K ANT, the top app receives 20K ANT. The next app gets paid 20% of the remaining pot. The remaining pot is 80K, and 20% of that is 16K ANT. This process continues until every app has been paid or the payout amount is bellow 200 ANT whichever comes first.
+
+**Blacklist**
+
+It's possible that the KPI mechanism can be gamed, for example the popularity metric could be manipulated by creating a bunch of organization and installing a bunch of otherwise unpopular applications. For the purposes of our initial pilot, rather than try an engineer un-gameable metric, we propose that we can rely on the community to report potential abuse and the discretion of Pinion to manage a blacklist for the purpose of excluding recipients that are suspected to be trying to maliciously manipulate payout amounts. In the future, this role could be decentralized by leveraging the Aragon Court.
+
+**Initial and Ongoing Execution**
+
+This proposal includes a one-time 50K DAI budget for Pinion to execute on the development and documentation effort related to surfacing these metrics to the community and preparing the initial launch of the app mining program.
+
+If approved, Pinion would:
+    - Work with [Apiary](apiary.1hive.org) to create a dashboard that includes these metrics and make the Aragon App Score publicly and easily accessible to the Aragon community.
+    - Provide documentation via PRs to hack.aragon.org for application developers to participate in the program
+    - Assist the AA and/or teams responsible for Aragon communications in preparing communications for the launch of Aragon's App Mining program
+    - Coordinate with the AA to execute payments each quarter to eligible recipients based on calculated Aragon App Scores
```

Comment:
Will these ongoing administrative duties require a budget after the initial 50k DAI is exhausted?

Reply:
I think that after the initial launch it may make sense for a different entity to take over, such as the AA, but the ongoing overhead should be minimal and can be allocated in a future AGP if necessary.